eslintrc.js

```diff
@@ -0,0 +1,121 @@
+module.exports = {
+  env: {
+    browser: true,
+    es6: true,
+    node: true,
+    'jest/globals': true,
+  },
+  extends: [
+    // 'eslint:recommended',
+    // 'plugin:import/recommended',
+    // 'plugin:react/recommended',
+    // 'plugin:@typescript-eslint/eslint-recommended',
+    // 'plugin:@typescript-eslint/recommended',
+    'prettier',
+    'prettier/react',
+    'prettier/@typescript-eslint',
+  ],
+  parser: '@typescript-eslint/parser',
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true,
+    },
+    project: ['tsconfig.json', 'cypress/tsconfig.json'],
+    sourceType: 'module',
+  },
+  settings: {
+    react: {
+      version: 'detect',
+    },
+  },
+  plugins: [
+    '@typescript-eslint',
+    '@typescript-eslint/tslint',
+    'jest',
+    'react',
+    'prettier',
+    'react-hooks',
+  ],
+  rules: {
+    '@typescript-eslint/class-name-casing': 'error',
+    '@typescript-eslint/indent': 'off',
+    '@typescript-eslint/interface-name-prefix': 'error',
+    '@typescript-eslint/member-delimiter-style': 'off',
+    '@typescript-eslint/type-annotation-spacing': 'off',
+    'arrow-parens': ['off', 'as-needed'],
+    'default-case': 'error',
+    'dot-notation': 'error',
+    'eol-last': 'off',
+    'guard-for-in': 'error',
+    'linebreak-style': 'off',
+    'new-parens': 'off',
+    'newline-per-chained-call': 'off',
+    'no-caller': 'error',
+    'no-debugger': 'error',
+    'no-empty': 'error',
+    'no-empty-function': 'error',
+    'no-eval': 'error',
+    'no-extra-semi': 'off',
+    'no-fallthrough': 'error',
+    'no-irregular-whitespace': 'off',
+    'no-multiple-empty-lines': 'off',
+    'no-new-wrappers': 'error',
+    'no-param-reassign': 'error',
+    'no-undef-init': 'error',
+    'no-unused-labels': 'error',
+    'no-use-before-define': 'warn', // Decide on this
+    'no-var': 'error',
+    'prefer-const': 'error',
+    'quote-props': 'off',
+    'space-before-function-paren': 'off',
+    'react/jsx-no-bind': 'warn', // Update
+    curly: 'error',
+    radix: 'error',
+    'prettier/prettier': [
+      'error',
+      {
+        arrowParens: 'avoid',
+        bracketSpacing: false,
+        jsxBracketSameLine: false,
+        printWidth: 100,
+        proseWrap: 'always',
+        semi: true,
+        singleQuote: true,
+        tabWidth: 2,
+        trailingComma: 'es5',
+        useTabs: false,
+      },
+    ],
+    // NOTE: Commented out everything below that caused problems. A lot of this is likely included in the stuff above.
```

Comment:
All of the stuff below is generated by https://github.com/typescript-eslint/tslint-to-eslint-config. A lot of it is likely covered with things above, and our goal should be to remove this section completely.

Some of the items below caused problems so I just commented them out for now.